vite.config.mts

```diff
@@ -215,6 +216,27 @@ export default defineConfig({
             template: 'treemap' // or 'sunburst', 'network'
           })
         ]
+      : []),
+
+    // Sentry sourcemap upload plugin
+    // Only runs during cloud production builds when all Sentry env vars are present
+    // Requires: SENTRY_AUTH_TOKEN, SENTRY_ORG, SENTRY_PROJECT env vars
```

Comment:
I don't love it, but want to put blanks for these into the `.env_example`?

Reply:
will do